montecarlo/cfast_mcarlo.py

```diff
@@ -225,9 +225,18 @@ def _section_windows(self):# {{{
 
         return "\n".join(txt)+"\n" if len(txt)>1 else ""
 # }}}
+
     def _section_doors_and_holes(self):# {{{
         ''' Randomize how doors are opened/close. '''
 
+        pre_evac = self.conf['pre_evac']
+        try:
+            if not pre_evac['1st']:
+                raise KeyError
+            first_percentile = pre_evac['1st']
+        except KeyError:
+            first_percentile = round(lognorm_percentiles_from_params(pre_evac['mean'], pre_evac['sd'])[0], 1)
+
```

Comment:
it wouldn't be enough to 
try: 
first_percentile = pre_evac['1st']
except KeyError:

Reply:
Yup, lines 234-235 seems redundant. Thanks!